client/src/main/java/io/pravega/client/stream/impl/ClientFactoryImpl.java

```diff
@@ -217,6 +217,22 @@ public ClientFactoryImpl(String scope, Controller controller, ConnectionPool con
         return createRevisionedStreamClient(getSegmentForRevisionedClient(scope, streamName), serializer, config);
     }
 
+    @Override
+    public <T> RevisionedStreamClient<T> createRevisionedStreamClient(String streamName, Controller controllerObj,
+                                                                      Serializer<T> serializer,
+                                                                      SynchronizerConfig config,
+                                                                      AccessOperation accessOperation) {
```

Comment:
also, for someone passing the controller client implementation, should this not require AccessOperation like its overloaded method. 

also one nit: please move this method below the other one.. so when users list the methods the non controller obj method shows up first as that is what the user apps will use.

Reply:
Resolved.